### ISSUE TITLE
Add Primitive Names

### ISSUE THREAD

**Issue by RichardTea** [KhronosGroup/glTF#2006]
Currently a mesh has a name but the primitives that make it up do not.
This can make it difficult to identify important primitives for re-assigning materials etc.

Similar use case as for issue #1036 

As is the case with "name" elsewhere, this would be informative for artists and not a reference within the glTF file itself.

At present I'm using an "extras" value for this in my implementation.

**Comment by donmccurdy** [comment]
I agree this would be useful; IMHO it lands in the same category as morph target names — not urgent enough to provide as an extension, but probably a no-brainer to include in the next glTF major version, whenever that might be.

In the meantime the same solution used for morph target names would also work here, many implementations look for a `"targetNames": ["name1", "name2", ...]` array in "extras" and decorate morph targets with human-readable names when available.
